grammars/blade.cson

```diff
@@ -11,21 +11,25 @@
     'patterns': [
       {
         'begin': '{{--'
-        'captures':
+        'beginCaptures':
           '0':
-            'name': 'punctuation.definition.comment.blade'
+            'name': 'punctuation.definition.comment.begin.blade'
         'end': '--}}'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.comment.end.blade'
         'name': 'comment.block.meta.embedded.blade'
       }
       {
         'begin': '(?<!@){{{'
-        'captures':
+        'beginCaptures':
           '0':
-            'name': 'support.punctuation.blade'
-          '1':
-            'name': 'source.php'
```

Comment:
I've restored the capturing groups, but you should perhaps consider adding a comment to elaborate on why they're necessary. Future contributors may also misinterpret those groups as a mistake.